packages/backend/src/api-impl.ts

```diff
@@ -42,6 +44,10 @@ export class BootcApiImpl implements BootcApi {
     this.webview = webview;
   }
 
+  async getExamples(): Promise<ExamplesList> {
+    return examplesCatalog as ExamplesList;
```

Comment:
Could use a test. I know it's not testing much, but just to confirm that we get examples loaded here.

Maybe could confirm some basic hygiene like 'all examples have descriptions with length > 20' or repo/image strings look valid?